### PR TITLE
WIP: Flake8 & isort

### PR DESCRIPTION
**Pull request by apollo13**
I copied most of the settings from what we have in Django

### CODE REVIEW

File: circuits/web/parsers/multipart.py [hunk 1]
```diff
@@ -32,34 +32,18 @@
     THE SOFTWARE.
 
 '''
-
-__author__ = 'Marcel Hellkamp'
```
Comment (spaceone) [inline circuits/web/parsers/multipart.py]: Hm, this is just a package from upstream.

**Comment by apollo13** [comment]
So, the testsuite passes and I think I've addressed all comments  by @spaceone -- any objections to merging that? (Integrating web.url as dependency is imo out of scope for this issue).

**Comment by spaceone** [comment]
You ignored some of my comments about the .decode('unicode_escape') changes

**Comment by apollo13** [comment]
@spaceone is that not addressed by https://github.com/circuits/circuits/pull/200/commits/ae187e6df19459af0765c1e59990e84fa5ebad98 ?

**Comment by spaceone** [comment]
No, circuits.web.http is affected

**Comment by spaceone** [comment]
And circuits.web.parsers.http

**Comment by apollo13** [comment]
Mhm, not sure I follow -- I've changed `circuits.web.parser.http`, but I think I've kept the original behavior of `bytes_to_str` there. The tests seem also okay (aside from changing them to adhere to the behavior change in b()).

Please tell me what I am missing

File: circuits/core/pollers.py [hunk 2]
```diff
@@ -57,7 +59,7 @@ def __init__(self, channel=channel):
     def _create_control_con(self):
         if platform.system() == "Linux":
             return os.pipe()
-        server = create_socket(AF_INET, SOCK_STREAM)
+        server = socket(AF_INET, SOCK_STREAM)
```
Comment (apollo13) [inline circuits/core/pollers.py]: Nope, no idea why the `socket` class was aliased to `create_socket` :D

File: tests/app/test_daemon.py [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
+from time import sleep
 
+import pytest
 
 from . import app
 
+if pytest.PLATFORM == "win32":
+    pytest.skip("Unsupported Platform")
+
+
```
Comment (apollo13) [inline tests/app/test_daemon.py]: Fixed, that said there is plenty of "too much whitespace" still around, but I am not going to fix all of those manually for this PR

File: circuits/node/utils.py [hunk 4]
```diff
@@ -13,7 +13,7 @@
 def load_event(s):
     data = json.loads(s)
 
-    name = bytes_to_str(data["name"].encode("utf-8"))
+    name = data["name"]
```
Comment (apollo13) [inline circuits/node/utils.py]: So the current code is: https://github.com/circuits/circuits/pull/200/files#diff-015e99aa1b430ebfaa9aa46322a22df0R16 -- I think that fixes it

Essentially the idea is:
 * On Py3 classnames are unicode/text which is what json returns, so there is nothing to do
 * On Py2 classnames need to be bytes so we encode via utf-8

File: circuits/web/http.py [hunk 5]
```diff
@@ -22,22 +23,9 @@
...
-    from urlparse import urlparse, urlunparse  # NOQA
-
-
 MAX_HEADER_FRAGENTS = 20
 HTTP_ENCODING = 'utf-8'
 
-try:
-    unicode
-except NameError:
-    unicode = str
```
Comment (apollo13) [inline circuits/web/http.py]: `unicode` was not used anywhere in this file and I do not think it was added as "utility" for other modules.

File: circuits/web/parsers/http.py [hunk 6]
```diff
@@ -187,7 +177,9 @@ def execute(self, data, length):
                 else:
                     self.__on_firstline = True
                     self._buf.append(data[:idx])
-                    first_line = bytes_to_str(b("").join(self._buf))
+                    first_line = b("").join(self._buf)
+                    if PY3:
+                        first_line = str(first_line, 'unicode_escape')
```
Comment (apollo13) [inline circuits/web/parsers/http.py]: Okay, this takes a little bit -- lets step back:

What did `bytes_to_str(b("").join(self._buf))` originally do:
On py2 `bytes_to_str` is a no-op: https://github.com/circuits/circuits/blob/5797ea3769d8197a0ca3a4c7db898a740ce11a89/circuits/six.py#L299-L300
hence leaving it out is okay for python 2.

On Py3 it did decode the bytes using `unicode_escape`:
https://github.com/circuits/circuits/blob/5797ea3769d8197a0ca3a4c7db898a740ce11a89/circuits/six.py#L281-L282
which my `if PY3` should do.

(`bytes_to_str` is no longer part [or rather was never] of six, hence I removed it when updating six to upstream)

Arguably the code is broken (I am not sure that unicode_escape is the way to go here), but it is the same brokenness as before (or same correctness for that matter)

File: circuits/web/parsers/http.py [hunk 7]
```diff
@@ -313,7 +305,7 @@ def _parse_headers(self, data):
             return False
 
         # Split lines on \r\n keeping the \r\n on each line
-        lines = [bytes_to_str(line) + "\r\n"
+        lines = [(str(line, 'unicode_escape') if PY3 else line) + "\r\n"
```
Comment (apollo13) [inline circuits/web/parsers/http.py]: Same answer as above

File: circuits/web/parsers/http.py (hunk 6 above)
Comment (spaceone) [inline circuits/web/parsers/http.py]: Alright. Then it is wrong on python3.

**Pull request merged**